<ui version="4.0" >
 <class>ConferenceList</class>
 <widget class="QWidget" name="ConferenceList" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QTreeView" name="treeView" />
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="refreshButton" >
     <property name="text" >
      <string>刷新</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="instantConfButton" >
     <property name="text" >
      <string>创建即时会议</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
